src/components/SortBy/SortBy.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
+import classNames from 'classnames/bind';
 import styles from './SortBy.scss';
 
+const cx = classNames.bind(styles);
+
 class SortBy extends Component {
   constructor() {
```

Comment:
If you use constructor, you should alway use props as an argument and pass it to super: `constructor(props) { super(props) }`